### DISCUSSION TITLE
Unit-testing cross-mingw.rb using wine

### DISCUSSION THREAD

**Discussion by suetanvil** [mruby/mruby#5562] [Ideas]
Lately, I've been experimenting with using MinGW and Wine to develop for Windows on a *nix system.   As part of this, I was able to extend the mruby build scripts to run the unit tests (`enable_bintest` and `enable_tests`) using Wine.  All tests now succeed for me.

Should I submit this as a pull request?

**Comment by matz** [comment]
Yes, please.

**Reply by suetanvil** [reply]
Will do, once I've unbroken things.

**Comment by suetanvil** [comment]
I've submitted the pull request [here](https://github.com/mruby/mruby/pull/5563).
